src/dotnet/install.sh

```diff
@@ -4,44 +4,10 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 #
-# Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/dotnet.md
-# Maintainer: The VS Code and Codespaces Teams
-
-DOTNET_VERSION="${VERSION:-"latest"}"
```

Comment:
Same here, can we add back this variable? 

```
DOTNET_VERSION="${VERSION}"
```

Reply:
Done